examples/step-89/step-89.cc

```diff
@@ -594,12 +598,12 @@ namespace Step89
     // as well. This is because we iterate over each side of the non-matching
     // face seperately (similar to a cell
     // centric loop).
-    template <bool weight_neighbor,
+    template <bool weight_neighbor, // PM: I would make this an argument
```

Comment:
Currently, this is not possible because `ExternalFaceIntegrator` only has the function `.submit_value()` if it is a `FEFaceEval` object and not a `FEBoundaryCondition` or `FERemoteEval` object :(